CMakeLists.txt

```diff
@@ -79,11 +79,11 @@ endif()
 
 add_subdirectory(src)
 add_subdirectory(data)
+add_subdirectory(tools)
```

Comment:
@sunweaver Why? This will also intall it.

Reply:
libayatana-indictor3-tools (Debian package) needs this. It simply brings back into the build status of the autotools version that we had before.

Seriously, I find it very helpful having the indicator loader utility in my DEBs (and yes, I enable tests for the DEBs, so...).

Pretty please, ACK this.